### PR TITLE
Update to include v0.6.1 Ubuntu based DLRS release

### PR DESCRIPTION
**Pull request by bd-dean**
Documentation update.

### CODE REVIEW

File: dlrs/README.md [hunk 1]
```diff
@@ -7,16 +7,23 @@ The Deep Learning Reference Stack is an integrated, highly-performant open sourc
...
+### For Ubuntu based images:
+* [Tensorflow 1.15, Intel® MKL-DNN and Intel® AVX512-Deep Learning Boost (v0.6.1)](https://hub.docker.com/r/sysstacks/dlrs-tensorflow-ubuntu)
+* [TensorFlow 2.2.0 with oneDNN primitives, Intel® MKL, Intel® DLBoost and OpenVINO™ - Deep Learning Deployment Toolkit v2020.1 (TBB)](https://hub.docker.com/r/sysstacks/dlrs-tensorflow2-ubuntu)
+* [PyTorch and Intel® MKL (v0.6.1)](https://hub.docker.com/r/sysstacks/dlrs-pytorch-clearlinux)
 
 
-Please see the tags tab in dockerhub to find the versions listed above, valid tags for version 6 of DLRS are `latest` (default), `v0.6.0` and for the eigen versions `v0.6.0-oss`
+
+
+Please see the tags tab in dockerhub to find the versions listed above, valid tags for version 0.6 of DLRS are `latest` (default), `v0.6.0` and for the eigen versions `v0.6.0-oss`
```
Comment (DnPlas) [inline dlrs/README.md]: Can we add a note saying eigen versions are only available for clearlinux?

Comment (bd-dean) [reply dlrs/README.md]: Done :)

File: dlrs/clearlinux/releasenote.md [hunk 2]
```diff
@@ -60,28 +60,67 @@ The [dlrs-pytorch-clearlinux:v0.6.0-oss](https://hub.docker.com/r/sysstacks/dlrs
...
+
+
+## The Deep Learning Reference Stack Release built on Ubuntu
+
+### The Deep Learning Reference Stack TensorFlow 2.2.0 with oneDNN primitives, Intel® MKL, Intel® DLBoost and OpenVINO™ - Deep Learning Deployment Toolkit v2020.1 (TBB) built on Ubuntu
+
+The [dlrs-tensorflow2-ubuntu](https://hub.docker.com/r/sysstacks/dlrs-tensorflow2-ubuntu) version v0.6.1 includes:
+  * Ubuntu 20.04
+  * Runtimes (Python 3)
+  * TensorFlow 2.1.0 optimized using oneAPI Deep Neural Network Library (oneDNN) primitives and Intel® AVX-512 Deep Learning Boost (Formerly Intel® VNNI)
```
Comment (Meadosc) [inline dlrs/clearlinux/releasenote.md]: Tensorflow 2.1 or 2.2? The header references a different version

File: dlrs/clearlinux/releasenote.md [hunk 3]
```diff
@@ -60,28 +60,67 @@ The [dlrs-pytorch-clearlinux:v0.6.0-oss](https://hub.docker.com/r/sysstacks/dlrs
...
+  * Intel OpenVINO™ model server v2020.1
+  * Transformers - State-of-the-art Natural Language Processing for TensorFlow 2.1.0
```
Comment (Meadosc) [inline dlrs/clearlinux/releasenote.md]: Same as above, TF 2.2

File: dlrs/clearlinux/releasenote.md (hunk 2 above)
Comment (bd-dean) [reply dlrs/clearlinux/releasenote.md]: good catch -- changing all to 2.1 for Ubuntu.

File: dlrs/clearlinux/releasenote.md (hunk 3 above)
Comment (bd-dean) [reply dlrs/clearlinux/releasenote.md]: latest commit aligns on tf 2.1

**Review by Meadosc — APPROVED** [review]
LGTM

**Pull request merged**